<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlexLens</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="classifyTab">
       <attribute name="title">
        <string>Klassifizierung</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="InputPanel" name="inputClassifyPanel" native="true"/>
        </item>
        <item>
         <widget class="ParameterPanel" name="parameterClassifyPanel" native="true"/>
        </item>
        <item>
         <widget class="OutputClassifyPanel" name="outputClassifyPanel" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="trainingTab">
       <attribute name="title">
        <string>Transfer Learning</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="InputPanel" name="inputTrainingPanel" native="true"/>
        </item>
        <item>
         <widget class="ParameterPanel" name="parameterTrainingPanel" native="true"/>
        </item>
        <item>
         <widget class="OutputClassifyPanel" name="outputTrainingPanel" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHilfe"/>
   <addaction name="actionStatistiken"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStatistiken">
   <property name="icon">
    <iconset resource="Icon/icon.qrc">
     <normaloff>:/icon/iconStat.png</normaloff>:/icon/iconStat.png</iconset>
   </property>
   <property name="text">
    <string>Statistiken</string>
   </property>
   <property name="toolTip">
    <string>Statistiken</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="icon">
    <iconset resource="Icon/icon.qrc">
     <normaloff>:/icon/iconQuestion.png</normaloff>:/icon/iconQuestion.png</iconset>
   </property>
   <property name="text">
    <string>Hilfe</string>
   </property>
   <property name="toolTip">
    <string>Hilfe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ParameterPanel</class>
   <extends>QWidget</extends>
   <header location="global">parameterpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputPanel</class>
   <extends>QWidget</extends>
   <header location="global">inputpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputClassifyPanel</class>
   <extends>QWidget</extends>
   <header location="global">outputclassifypanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
